editor-server/src/resolvers/dancer.ts

```diff
@@ -85,61 +90,61 @@ export class DancerResolver {
     @Ctx() ctx: TContext
   ) {
     const { id, name } = newDancerData;
-    const newDancer = await ctx.db.Dancer.findOneAndUpdate(
-      { id },
-      { name },
-      { new: true }
-    ).populate("parts");
+    const newDancer = await ctx.prisma.dancer.update({
+      where: { id },
+      data: { name },
+      include: { parts: true },
+    });
     if (newDancer) {
       await initRedisControl();
```

Comment:
In new protocol, there is no need to init Redis here. (since name is not included in control frame)